tests/pr/milestone.test.ts

```diff
@@ -0,0 +1,117 @@
+import { checkMilestone as milestone } from "../org/pr/milestone";
```

Comment:
Unfortunately, the changes to this file were too many and with the move it just looks like a completely new file in the diff.

That might not be a bad things, actually. Gives a better chance to review it.

Unlike the production code, this one is a bit different, due to the way we need to write `Dangerfile`s by importing from the `node_modules/` at runtime and therefor mock in the `global` context in the tests.